awx/main/utils/handlers.py

```diff
@@ -91,6 +91,23 @@ def result(self):
         return self
 
 
+class RotatingProductionLogHandler(logging.handlers.RotatingFileHandler):
+
+    @property
+    def level(self):
+        from django.conf import settings
+        try:
+            return logging._nameToLevel[settings.LOG_AGGREGATOR_LEVEL]
+        except Exception:
+            return 30  # WARNING
```

Comment:
This is still really different from

https://github.com/ansible/awx/blob/92facc670630d622a2b953aa02d00fef38362ea8/awx/main/utils/filters.py#L94-L96

If we're going with this pattern, then the above blurb should be moved into `handlers.py` as well. This logic should be shared completely between `RotatingProductionLogHandler` and `AWXProxyHandler`, because the point of the request was unification between the two. This could be done as a mixin or some shared method.

I don't love the handler level (as opposed to the filter), but I'm more indifferent than anything. I just want the option selected to be used uniformly.

Reply:
👍